community_images/mysql/ironbank/image.yml

```diff
@@ -33,3 +33,5 @@ runtimes:
       environment:
         MYSQL_ROOT_PASSWORD: my_root_password
         MYSQL_ROOT_HOST: "%"
+      volumes:
+        assets: /usr/share/zoneinfo
```

Comment:
Do you think we can add this to rfignore?

Reply:
Mysql was failing because `/usr/share/zoneinfo` directory was not present and `mysql` user have no access to create it. So, the option i had was to add a empty volume/folder and empty folder is not picked by git. So, here i made it to point to `assets` folder. I think adding it to `.rfignore` will not work.